src/protocols/aeternity/libs/AeternityAdapter.ts

```diff
@@ -304,6 +304,8 @@ export class AeternityAdapter extends BaseProtocolAdapter {
         ...transaction,
         pending: true,
         protocol: PROTOCOLS.aeternity,
+        microTime: Date.now(),
```

Comment:
Why do we need to add `microTime` to pending transactions?

Reply:
This was our previous approach to show that it was done some time ago.

Reply:
But it seems that `microTime` for the custom pending transaction is not being set :(. And the whole logic is not working.

Reply:
So maybe this needs another fix?

Reply:
Will do it within this PR.

Reply:
1. We should display the timestamp in the Latest transactions widget on the wallet Dashboard as we currently do because this way the user can know approximately how much time the transaction is in a pending state:
![Pending-3](https://github.com/superhero-com/superhero-wallet/assets/12225410/cef57e1c-c799-4f59-9be2-a96df2ff2895)

2. We should not display the timestamp in the Transactions tab list under Account details because we have limited UI space there and because the user can access the single transaction details screen to see the exact timestamp.

3. Add the timestamp for pending transaction on the Transactions details screen in case we can get it (the transaction is initiated by the user by the currently active wallet account): 
![Pending-5](https://github.com/superhero-com/superhero-wallet/assets/12225410/85c1fad7-ec0e-4b0d-bdea-955235aaa8a4)


@CedrikNikita what do you think?

Reply:
I suggest to move last part to another ticket and apply current fix.